README.md

```diff
@@ -1,30 +1,52 @@
-# Sourcegraph extension for Phabricator
+# Sourcegraph's Phabricator integration
 
 Get code intelligence on Phabricator with Sourcegraph.
 
 Check out our [docs site](https://about.sourcegraph.com/docs/features/phabricator-extension) for more information.
 
 ## [Installation](https://https://about.sourcegraph.com/docs/features/phabricator-extension#option-a-single-installation-script)
 
-1. To install Sourcegraph on your Phabricator instance run `./install.sh <path/to/phabricator/root> <sourcegraph url>` inside your Phabricator instance.
-2. Restart Phabricator instance.
+1. Clone this repository into a sourcegraph folder located in your Phabricator extension directory.
```

Comment:
This is confusing; from the POV of a user:

- "a sourcegraph folder" - does one already exist? will there be multiple (why not "the"?)?
- "in your Phabricator extension directory" - the directory is actually called `extensions`; is this different?

How about:

> 1. Clone this repository to `phabricator/src/extensions/sourcegraph` (inside your Phabricator installation directory).